config/versions/versions_test.go

```diff
@@ -0,0 +1,113 @@
+package versions
+
+import (
+	"context"
+	"testing"
+
+	"github.com/buger/jsonparser"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/thrasher-corp/gocryptotrader/common"
+)
+
+func TestDeploy(t *testing.T) {
+	t.Parallel()
+	m := manager{}
+	_, err := m.Deploy(context.Background(), []byte(``))
+	assert.ErrorIs(t, err, errNoVersions)
+
+	m.registerVersion(1, &TestVersion1{})
+	_, err = m.Deploy(context.Background(), []byte(``))
+	require.ErrorIs(t, err, errVersionIncompatible)
+
+	m = manager{}
+
+	m.registerVersion(0, &Version0{})
+	_, err = m.Deploy(context.Background(), []byte(`not an object`))
+	require.ErrorIs(t, err, jsonparser.KeyPathNotFoundError, "Should throw the correct error trying to add version to bad json")
+	require.ErrorIs(t, err, common.ErrSettingField, "Should throw the correct error trying to add version to bad json")
+	require.ErrorContains(t, err, "version", "Should throw the correct error trying to add version to bad json")
```

Comment:
```suggestion
	require.ErrorContains(t, err, "version", "Must throw the correct error trying to add version to bad json")
```

Reply:
Missed the case insensitive versions of this in my new check, soz.

Fixed https://github.com/gbjk/gocryptotrader/commit/ba1a21453